vllm/model_executor/layers/logits_processor.py

```diff
@@ -67,7 +67,7 @@ def _get_logits(self, hidden_states: torch.Tensor, embedding: torch.Tensor,
         if embedding_bias is not None:
             logits += embedding_bias
         # NOTE(kzawora): HPU PT bridge is missing support for single-rank gather. We'll use all-gather on Gaudi for now.
-        gather_op = tensor_model_parallel_all_gather if is_hpu() else tensor_model_parallel_gather
+        gather_op = tensor_model_parallel_gather
```

Comment:
why do we need this?

Reply:
we don't need this for this change, it's for 8x, I will remove it